pkg/crc/preflight/preflight_windows.go

```diff
@@ -50,6 +50,7 @@ var hypervPreflightChecks = []Check{
 		check:            checkHyperVInstalled,
 		fixDescription:   "Installing Hyper-V",
 		fix:              fixHyperVInstalled,
+		flags:            StartUpOnly,
```

Comment:
`fixHyperVInstalled` will never be used after this. This can be cleaned up in a follow-up commit.